### DISCUSSION TITLE
Material de las clases 3 y 17 de Julio

### DISCUSSION THREAD

**Discussion by gonzalemi** [dds-jv/foro#147] [Consultas generales] [answered]
Hola!

No estoy pudiendo encontrar en la bitácora los videos y presentación de las clases del 3 y 17 de Julio y me interesaría repasarlas ya que lamentablemente no pude asistir. Entiendo que ésta última es super reciente y si seguramente estén trabajando en ello, perdón el pedido tan pronto.

Muchas gracias.

**Comment by gastonprieto** [comment, marked as answer]
Buenas!

Te paso los links de los videos:

[clase del 17/7](https://us02web.zoom.us/rec/share/vPFZJZ7o-11OeM_f9mvUfa4KHZTIaaa80XMeqfdfzhsApPeGsjh_pQBjnOvnlinV?startTime=1595023516000)

[clase 3/7](https://us02web.zoom.us/rec/share/2Mxndr796nlJT53_yhGBRqIIApWieaa8hCAZ-vVcxEkhQbUa6sK2gu20_Mzp7GQ7?startTime=1593813686000)

**Comment by gonzalemi** [comment]
Hola Gastón, mil gracias por la pronta respuesta.
